Add 1.21.x Blocks

HerobrainGamer opened the pull request:
- Updated Dependencies.
- Fixed the required NPM vulnerabilities that are preventing the code from working.
- Removed extra comma in Spanish translation.
- Fix an error with translations "zh-Hans" & "zh-Hant" flag SVGs causing a build error.
- Fix Packed Mud texture encroaching on the Andesite Slab texture.
- Added support for MC 1.21, 1.21.2, & 1.21.4 blocks.
- Stone Colour added Pale Oak Log (Horizontal).
- White Wool Colour added Pale Oak Planks, Slab & Log (Vertical).
- Light Gray Wool Colour added Pale Moss Block & Carpet.
- Green Wool Colour added Moss Block & Carpet.
- Orange Terracotta Colour added Redstone Lamp, Block of Resin, Resin Bricks, & Resin Brick Slab.
- Gray Terracotta Colour added Tuff Slab.

HerobrainGamer committed:
Add 1.21.x Blocks

- Updated Dependencies.
- Fixed the required NPM vulnerabilities that are preventing the code from working.
- Removed extra comma in Spanish translation.
- Fix an error with translations "zh-Hans" & "zh-Hant" flag SVGs causing a build error.
- Fix Packed Mud texture encroaching on the Andesite Slab texture.
- Added support for MC 1.21, 1.21.2, & 1.21.4 blocks.
- Stone Colour added Pale Oak Log (Horizontal).
- White Wool Colour added Pale Oak Planks, Slab & Log (Vertical).
- Light Gray Wool Colour added Pale Moss Block & Carpet.
- Green Wool Colour added Moss Block & Carpet.
- Orange Terracotta Colour added Redstone Lamp, Block of Resin, Resin Bricks, & Resin Brick Slab.
- Gray Terracotta Colour added Tuff Slab.

0x4e49434f4c45 commented:
Hi - I just wanted to say thank you for taking the time to do this. I've deployed this version on a private server that my friends and I use. This repo doesn't look like it's been touched in a while, so I thought you might like to know your work is being used and appreciated :)

HerobrainGamer committed:
Add 26.2 Blocks

- Added Potent Sulfur to Gold
- Added Sulfur to Yellow Wool
- Added Cinnabar to Red Wool
- Also Added Lodestone & Pale Oak Leaves to Metal (Iron) as I forgot about them previously.
- Note there are no other really useful blocks between 1.21.4 and 26.2 that realistically needs to be added.
